examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -1634,8 +1636,16 @@ THREE.LegacyGLTFLoader = ( function () {
 									geometry.addAttribute( 'skinIndex', bufferAttribute );
 									break;
 
+								default:
+									var material = json.materials[ primitive.material ];
+									var attributeNames = json.techniques[ material.technique ].attributes;
+									var attributeName = Object.keys( attributeNames )[ attributeIndex ];
```

Comment:
This depends on `primitive.attributes` having the same order as `technique.attributes`, which is not guaranteed. Would this work?

```js
var attributeMap = invertObject( json.techniques[ material.technique ].attributes );
var attributeName = attributeMap[ attributeId ];


function invertObject ( input ){

  var output = {};

  for( var key in input ){

    output[ input[ key ] ] = key;

  }

  return output;

}
```

Reply:
Ugh, I see I've missed the indirection on the `parameters` array... I guess it is not quite this simple. I haven't got time right now but may need to take another look in a day or two. 😕

Reply:
Yes, that's right. I haven't looked at the spec to see if that's always the case, just saw it held in all the glTF files I looked at. If thet's really not guaranteed, then yeah, the complexity increases.

Reply:
I will attempt to update that to look through the attributes to find a match with the current attributeId.